idx/idx.go

```diff
@@ -13,6 +13,8 @@ import (
 	schema "gopkg.in/raintank/schema.v1"
 )
 
+const OrgIdPublic = 0
```

Comment:
shouldn't we just make this configurable with a command line arg?

Reply:
what's the benefit of that?
I think this should be a constant across the entire stack:
org 0 = public data
org 1 = admin org
org >1 = private tenant orgs

making it configurable seems to cause nothing but complications.

Reply:
> Wasnt the whole point of using a variable (or const) to remove the need to have special values.
No, the point was to only use orgid's that are >=0 so that they are encodeable as a uint.  this is also what was explained in the original ticket #260 

there's nothing wrong with special reserved values per se. I think conventions are useful, as long as they are simple and consistently applied. they simplify things (less config flags to worry about, clearer code)

Your proposed simplification for WP probes sounds nice, but does it really conflict with the proposed convention? is it not possible to just create that public org with id 0 ?

Reply:
I dont really like using the zero value, as it then makes it impossible to know if the value was set or not.

But as we dont have a use case for needing non-zero right now, lets just leave this. It will be trivial to change this later if we want/need to.